spec/visual/samples/racket

```diff
@@ -104,3 +104,4 @@
 
 ;; #true is not highlighted properly
 (define ht #true)
+(define hf #false)
```

Comment:
Should we also add [`#T` `#F` `#t` `#f`](https://docs.racket-lang.org/reference/reader.html#%28part._parse-boolean%29) samples here?

Reply:
Sure. Can do.